stories/Button.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
```

Comment:
Let's use Flow for all stories. This will be crucial in making sure the story code doesn't bit rot as the app evolves.

```diff
diff --git a/package.json b/package.json
index 49cbb374c..b79219d42 100644
--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
     "eslint": "eslint .",
     "flow": "flow",
     "flow:check": "flow check",
-    "flow:dev": "flow stop; flow start; chokidar .flowconfig flow/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
+    "flow:dev": "flow stop; flow start; chokidar .flowconfig flow/ stories/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
     "stylelint": "stylelint --syntax scss **/*.scss",
     "lint": "npm run eslint && npm run stylelint",
     "prettier": "prettier --write '**'",
diff --git a/stories/Badge.js b/stories/Badge.js
index 51863dfe0..493b02b42 100644
--- a/stories/Badge.js
+++ b/stories/Badge.js
@@ -1,3 +1,4 @@
+/* @flow */
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 
diff --git a/stories/Button.js b/stories/Button.js
index 0fa64a1c4..a1a49d80c 100644
--- a/stories/Button.js
+++ b/stories/Button.js
@@ -1,3 +1,4 @@
+/* @flow */
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 
```

Reply:
good idea 👍  I will add!